keras_trainer/data_generators.py

```diff
@@ -431,6 +456,7 @@ class EnhancedDataFrameIterator(EnhancedBatchFromFilesMixin, EnhancedIterator):
         x_col: string, column in `dataframe` that contains the filenames (or
             absolute paths if `directory` is `None`).
         y_col: string or list, column/s in `dataframe` that has the target data.
+        z_col: list or  none, column in `dataframe` that has the custom crop coordinates.
```

Comment:
What do you mean by "none"? The column in `dataframe` can be empty, but `z_col` will always be a list (of values or empty), right?

```suggestion
        z_col: list, (optional) column in `dataframe` that contains the custom crop coordinates.
```